hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -1377,6 +1382,33 @@ private static String getObjectName(ModelNode modelNode, OperationContext contex
         }
     }
 
+    private static List<OperationParam> getOpParamListFromOpNode(ModelNode modelNode) throws OperationFailedException {
+
+        List<OperationParam> ret = new ArrayList<>();
+
+        ModelNode params = modelNode.get("param");
+        if (params.getType()==ModelType.UNDEFINED) {
+            return Collections.emptyList();
+        }
+        List<Property> ps = params.asPropertyList();
+        for (Property prop : ps) {
+            String name = prop.getName();
+            ModelNode m = prop.getValue();
+            String typ = m.get("type").asString();
```

Comment:
do not hardcode "type" - use the constant in the Attribute class. need to use the Attribute class and resolve expressions. See "getString" usages as examples